docs/requirements.md

```diff
@@ -0,0 +1,192 @@
+# Requirements
+
+This document describes the requirements. 
+
+The favored usage of eulabeia is via [docker](#Docker).
+
+It is divided between two methods:
+- [docker](#Docker), will describe how to setup docker and use the provided makefile
+- [local](#Local), will describe how to setup and install the requirements without provided images
+
+This document will use Debian 11 as a basis and is targeting developer machines with relatively lax security requirements. 
+
+For production usage please harden docker, redis and mosquitto accordingly.
+
+## Docker
+
+To use eulabeia with docker you need to install `go`, `docker` as well as `gnu-make` to make it easier to use you can add your user to the docker group.
+
+```
+# apt-get install docker.io make golang
+# usermod -a -G docker your_username
+```
+
+To reload the newly assign group, login to your user account again
+
+```
+$ su - $USER
+$ docker ps
+```
+
+to build the images based on your changes you can use `make start-container` with that you have a useable sensor, director and broker up and running.
+
+If you want to verify the setup you can simply use `make run-example-client`.
+
+If you have `mosquitto-clients` installed, you can can verify that the local connection works:
+```
+$ mosquitto_sub -L 'mqtt://localhost:9138/scanner/#'
+```
+
+to subscribe to any message inside the scanner context. 
+
+Afterwards you can send a irregular message
+
+```
+$ mosquitto_pub -L 'mqtt://localhost:9138/scanner/scan/cmd/director' -m "hi"
+```
+
+### Broker
+
+To run a broker you can execute `make start-broker` this will run mosquitto and open a local port 9138.
+
+### Sensor
+
+To run a sensor you can execute `make start-sensor` this will run mosquitto and open a local port 9138 as well as start a sensor containing a test nasl file (oid: `1.3.6.1.4.1.25623.1.0.90022`) used by openvas.
+
+To communicate with the sensor use mqtt as described previously.
+
+### Director
+
+To run a director you can execute `make start-director` this will run mosquitto and open a local port 9138 as well as start a director.
```

Comment:
```suggestion
To run a director you can execute `make start-director`. This will run the mosquitto broker and opens the local port `9138` as well as start a director.
```